index.html

```diff
@@ -3782,6 +3809,165 @@ <h2>
           </tr>
         </table>
       </section>
+      <section data-dfn-for="MerchantValidationEvent" data-link-for=
+      "MerchantValidationEvent">
+        <h2>
+          <dfn>MerchantValidationEvent</dfn> interface
+        </h2>
+        <pre class="idl">
+          [Constructor(DOMString type, optional MerchantValidationEventInit eventInitDict),
+          SecureContext, Exposed=Window]
+          interface MerchantValidationEvent : Event {
+            readonly attribute USVString validationURL;
+            void complete(Promise&lt;any&gt; merchantSessionPromise);
+          };
+        </pre>
+        <section>
+          <h3>
+            <dfn data-lt=
+            "MerchantValidationEvent.MerchantValidationEvent()"><code>MerchantValidationEvent</code>
+            Constructor</dfn>
+          </h3>
+          <p data-tests=
+          "MerchantValidationEvent/constructor.https.html, MerchantValidationEvent/constructor.http.html">
+            The <dfn>event constructing steps for a
+            <code>MerchantValidationEvent</code></dfn>, which take a
+            <a>MerchantValidationEvent</a> <var>event</var>, are as follows:
+          </p>
+          <ol class="algorithm">
+            <li>Let <var>base</var> be the <a data-cite=
+            "dom#context-object">event</a>’s <a data-cite=
+            "!html/multipage/webappapis.html#relevant-settings-object">relevant
+            settings object</a>’s <a data-cite=
+            "!html/multipage/webappapis.html#api-base-url">API base URL</a>.
+            </li>
+            <li>Let <var>input</var> be the empty string.
+            </li>
+            <li>If <var>eventInitDict</var> was passed, set <var>input</var> to
+            the value of <var>eventInitDict</var>["<a>validationURL</a>"].
+            </li>
+            <li data-link-for="MerchantValidationEventInit">Let
+            <var>validationURL</var> be the result of <a data-cite=
+            "!url#concept-url-parser">URL parsing</a> <var>input</var> and
+            <var>base</var>.
+            </li>
+            <li>If <var>validationURL</var> is failure, throw a
+            <a>TypeError</a>.
```

Comment:
Needs to initialize _event_'s `validationURL` attribute to _validationURL_

Reply:
D'oh! Added just below.